integrationtest/integration_test.go

```diff
@@ -268,63 +364,39 @@ func TestAuthAliasMetadataAssignment(t *testing.T) {
 		t.Fatalf("Expected successful login but got: %v", err)
 	}
 
-	// query the entity alias and match up its custom metadata
-	secret, err := client.Logical().List("identity/entity-alias/id")
+	// verify that the templated policy works by creating key value pairs at root/data/foo with the kubernetes auth token
+	kv1MountCleanup := setupKv1Mount(t, client, kvPath)
+	defer kv1MountCleanup()
```

Comment:
nit: I would move this call up just before creating the policy for better readability, keeping the most closely related bits closer together.